### ISSUE TITLE
Visibility training

### ISSUE THREAD

**Issue by MiraPurkrabek** [pengzhansun/OPORT#2]
Hello @pengzhansun,

thanks for the good works. Could you please elaborate more on visibility training setup? From the paper, it seems your accuracy is almost 90%. How do you measure it and train it?

Thank you for your response.
